src/utils/devtools.ts

```diff
@@ -29,61 +33,68 @@ export const devtools = <T extends object>(proxyObject: T, name?: string) => {
 
   let isTimeTraveling = false
   const devtools = extension.connect({ name })
-  const unsub1 = subscribe(proxyObject, () => {
+  const unsub1 = subscribe(proxyObject, (ops) => {
+    const action = ops
+      .filter(([_, path]) => path[0] !== DEVTOOLS)
+      .flatMap(([op, path]) => `${op}:${path.map(String).join('.')}`)
+      .join(', ')
+
+    if (!action) {
+      return
+    }
+
     if (isTimeTraveling) {
       isTimeTraveling = false
     } else {
+      const proxyWithoutDevtools = Object.assign({}, snapshot(proxyObject))
+      delete (proxyWithoutDevtools as any)[DEVTOOLS]
+
       devtools.send(
-        `Update - ${new Date().toLocaleString()}`,
-        snapshot(proxyObject)
+        `${action} - ${new Date().toLocaleString()}`,
+        proxyWithoutDevtools
       )
```

Comment:
I feel like we have a better way to show an action. At least, this would be good.
```js
devtools.send({
  type: action,
  updatedAt: new Date().toLocaleString(),
})
```

We could make the `action` string more compact and put full infor in the object. what do you think?

Reply:
That one is better! Done.